homedecor_doors_and_gates/init.lua

```diff
@@ -183,12 +183,12 @@ for _, door in ipairs(door_list) do
 
 	if door.alpha then
 		local def = table.copy(minetest.registered_nodes[nn_a])
-			def.use_texture_alpha = true
-			def.mesh = "door_a.obj"                -- leaving this out will break the _a model
+			def.use_texture_alpha = "blend"
+			--def.mesh = "door_a.obj"                -- leaving this out will break the _a model
```

Comment:
So it can be removed?
I've no idea what that comment is talking about but I guess you also have no idea and therefore just commenting it out instead of removing?

Reply:
i have since did some digging and talking with another member, turns out it was a legacy hack to fix a issue. the reason it needs to be removed as mtg has moved from a .obj file to a .b3d file. the whole code in this section is a mess, but within this PR i am just fixing warnings/errors

Reply:
https://github.com/mt-mods/homedecor_modpack/pull/20/commits/e8f7a3efea8fbe8d48be77c6404833d8715bee4c resolved